Tutorials/ASP.NET/WebApi/README.md

```diff
@@ -0,0 +1,119 @@
+
+1. Open Visual Studio and create a new project.
+
+2. Search for the **ASP.NET Core Web Application** template. Click **Next** to proceed.
+
+3. Set the project name and click **Create**.
+
+4. Select **ASP.NET Core 5.0** and **API** and click Create.
+
+For more information, see the following:
+
+  * [Create web APIs with ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/web-api)
+  
+## Step 2: Add the DevExpress XPO Package
+
+Use any of the following approaches to install the XPO package:
+
+**Package Manager Console**
+
+* From the Visual Studio **Tools** menu, select **NuGet Package Manager > Package Manager Console**.
+* Make sure **Package source** is set to **All**, **nuget.org** or **DevExpress 20.1 Local** and run the following command: 
+    ```console
+  Install-Package DevExpress.Xpo
+  ```
+
+**NuGet Package Manager**
+
+* In the **Solution Explorer**, right-click the project name and select **Manage NuGet Packages**.
+* Make sure **Package source** is set to **All**, **nuget.org** or **DevExpress 20.1 Local** on the right.
+* In the left top corner under **Browse**, search for the `DevExpress.Xpo` package and select its latest version on the right.
+* Click **Install** on the right and accept all the licenses in the **License Acceptance** dialogs.    
+
+For more information, see the following:
+
+  * https://www.nuget.org/
+  * https://nuget.devexpress.com/
+
+## Step 3: Register DevExpress.Xpo.DB.WebApiDataStoreService
+
+1. Open the *apsettings.json* file and add a connection string to a root element.
+   ```json
+    {
+        ...
+        "ConnectionStrings": {
+            "YourConnectionStringName": "data source=(localdb)\mssqllocaldb;initial catalog=DxSample;user id=sa;password=dx"
+        } 
+    }
+    ```
+2. Open the *Startup.cs* file and add the following code to the **ConfigureServices** method.
+    ```cs
+    string connectionString = Configuration.GetConnectionString("YourConnectionStringName");
+    IDataStore dataStore = XpoDefault.GetConnectionProvider(connectionString, AutoCreateOption.SchemaAlreadyExists);
+    WebApiDataStoreService service = new WebApiDataStoreService(dataStore);
+    services.AddSingleton(service);
+    ```
+3. Add XML formatters, because WebApiDataStoreClient sends data in the XML format.
+    ```cs
+    services.AddMvc().AddXmlSerializerFormatters();
+    ```
+3. Add the CORS policy to allow access from a different origin. You can add multiple origins. For each origin, enable the POST method and the "Content-Type" header.
+    ```cs
+    services.AddCors(options =>
+        options.AddPolicy("XPO", builder =>
+            builder.WithOrigins("https://localhost:44317")
+                .WithMethods("POST")
+                .WithHeaders("Content-Type")));
+    ```
+
+## Step 4: Add a Web API controller
+
+1. In the **Solution Explorer**, select the **Controllers** folder and create the *XpoController.cs* file (the file and class names are up to you).
```

Comment:
```suggestion
1. In the **Solution Explorer**, select the **Controllers** folder and create the *XpoController.cs* file (you can specify the file and class names).
```

Reply:
I wanted to change this part (inside parentheses) to "you can use another file name), but finally I remove it because this part is not important.

Reply:
I also think "In the **Solution Explorer**, select the **Controllers** folder and create the *XpoController.cs* file" is sufficient.